template/types/media.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module '*.png';
+declare module '*.jpg';
+declare module '*.jpeg';
+declare module '*.svg';
+declare module '*.gif';
```

Comment:
Interesting, hadn't seen that before.

Reply:
I've forgotten where I got it from, but I needed it in a past project 🤷‍♂️